src/lv_interop.h

```diff
@@ -40,10 +41,9 @@ namespace grpc_labview
         }
 
         bool IsValid() { return true; }
```

Comment:
The intent of the IsValid on gRPC ID was to eventually have a managed list of all pointers that we valid.  The token_manager seems like the right thing to store all of the valid pointers.  IsValue can get the manager and return false if the pointer is not registered.  All of the entrypoints should be using CastTo and handling the null case as an error returned to LabVIEW.  I would put the use of the token manager here.

Reply:
It is safe to call IsValid.  It is not a virtual function and C++ does not care if the this pointer is valid or not.  So you can call IsValid and that can call into the token manager to ensure the pointer is still valid (not deleted, not null).